docker/script/bootstrap.sh

```diff
@@ -2,6 +2,12 @@
 
```

Comment:
this file shouldnt have any changes. the current code in v1.10.15 branch is copied from v1.10.12. we should keep that the same. The diff is showing changes from 2.0 being merged in.

checkout the v1.10.15 branch on your fork and apply the changes that are version specific.